container_accessibility/tests/test_user_limit.py

```diff
@@ -0,0 +1,20 @@
+from odoo.exceptions import UserError
+from odoo.tests.common import TransactionCase
+from odoo.tools import config
+
+
+class TestUserLimit(TransactionCase):
+    def test_exceeded(self):
+        config["user_limit"] = "5"
+        with self.assertRaises(UserError):
+            for i in range(10):
```

Comment:
Perhaps style, but it is not clear to me here why you iterate to 10. Why not first create 5 users, then test that creating the 6th raises an error? In theory it is possible that another setting causes this limit as well, so to be correct, you could also test that you are in fact allowed to create the user if you increase the limit.

Reply:
I've created this test just to increase the coverage for now. By my priorities and limited time I can't create proper tests ATM.